app/controllers/green_lanes/check_your_answers_controller.rb

```diff
@@ -0,0 +1,41 @@
+module GreenLanes
+  class CheckYourAnswersController < ApplicationController
+    include GreenLanesHelper
+
+    before_action :check_green_lanes_enabled,
+                  :disable_switch_service_banner,
+                  :disable_search_form
+    before_action :goods_nomenclature, only: :new
+    before_action :set_instance_variables, only: :new
+
+    def new
+      @category_one_assessments = determine_category.cat1_with_exemptions
+      @category_two_assessments = determine_category.cat2_with_exemptions
+      @ans = params[:ans]
+      @c1ex = params[:c1ex]
+      @c2ex = params[:c2ex]
+    end
+
+    private
+
+    def goods_nomenclature
+      @goods_nomenclature ||= GreenLanes::FetchGoodsNomenclature.new(results_params).call
+    end
+
+    def set_instance_variables
+      @commodity_code = results_params[:commodity_code]
+      @country_of_origin = results_params[:country_of_origin] || GeographicalArea::ERGA_OMNES
+      @country_description = GeographicalArea.find(@country_of_origin).description
+      @moving_date = results_params[:moving_date]
+    end
+
+    def determine_category
+      @determine_category ||= DetermineCategory.new(@goods_nomenclature)
+    end
+
+    def results_params
```

Comment:
I was trying to think of a better name,
it is called result_params,
but the params do not mention result at all :thinking:

Reply:
erm i think this is fine as they are going to the results controller? but it has the params to build the result? Answers?

Reply:
I guess he's thinking this because the idiom in controllers is that receiving params methods are typically called `<model_name>_params` and it makes you think you're in the results controller. I don't mind too much personally.